examples/ipv6only-app/esp32/README.md

```diff
@@ -0,0 +1,98 @@
+# CHIP ESP32 IPV6 Only Example Application
+
+This application implements ESP32 wifi control to support IPV6 tests.
+
+Once connected the application acts as a UDP echo server and will echo udp
+messages it receives, this can be used to test for diconnect events.
+
+-   [CHIP ESP32 IPV6 Only Example Application](#chip-esp32-ipv6-only-example-application)
+    -   [Building the Example Application](#building-the-example-application)
+        -   [To build the application, follow these steps:](#to-build-the-application-follow-these-steps)
+    -   [Testing the Example Application](#testing-the-example-application)
+
+---
+
+## Building the Example Application
+
+Building the example application requires the use of the Espressif ESP32 IoT
+Development Framework and the xtensa-esp32-elf toolchain.
+
+The VSCode devcontainer has these components pre-installed, so you can skip this
+step. To install these components manually, follow these steps:
+
+-   Clone the Espressif ESP-IDF and checkout release/v4.1 branch
+
+          $ mkdir ${HOME}/tools
+          $ cd ${HOME}/tools
+          $ git clone https://github.com/espressif/esp-idf.git
+          $ cd esp-idf
+          $ git checkout release/v4.1
+          $ git submodule update --init
+          $ export IDF_PATH=${HOME}/tools/esp-idf
+          $ ./install.sh
+
+-   Install ninja-build
+
+          $ sudo apt-get install ninja-build
+
+### To build the application, follow these steps:
+
+Currently building in VSCode _and_ deploying from native is not supported, so
+make sure the IDF_PATH has been exported(See the manual setup steps above).
+
+-   In the root of the example directory, sync the CHIP tree's submodules and
+    source `idf.sh`. Note: This does not have to be repeated for incremental
+    builds.
+
+          $ source idf.sh
+
+-   Configuration Options
+
+        To choose from the different configuration options, run menuconfig
+
+          $ idf.py make menuconfig
+
+        This example uses UART0 for serial communication. You can change this through
+        `PW RPC Example Configuration`. As a result, the console has been shifted to UART1
+        You can change this through `Component config` -> `Common ESP-related` ->
+        `UART for console output`
+
+        To use the default configuration options, run the default config
+
+          $ idf.py make defconfig
```

Comment:
You can remove this. Also, can you please refer the latest updated documentation for the ESP32 examples (README.md).

Reply:
done